client/gov/cli/query.go

```diff
@@ -287,22 +287,16 @@ func GetCmdQueryGovConfig(storeName string, cdc *wire.Codec) *cobra.Command {
 			}
 
 			if keyStr != "" {
-				res, err := ctx.QueryStore([]byte(keyStr), storeName)
+				parameter.RegisterGovParamMapping(&govparams.DepositProcedureParameter,
+					&govparams.VotingProcedureParameter,
+					&govparams.TallyingProcedureParameter,)
+
```

Comment:
Move 'parameter.RegisterGovParamMapping' into a utils.go. And should be reused in WriteFile().

Reply:
Thank you . Give me  a lot of chances of aading the commits